packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -227,6 +241,8 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
             onRenderMessage={onRenderMessage}
             /* @conditional-compile-remove(file-sharing) */
             onRenderFileDownloads={onRenderFileDownloads}
+            /* @conditional-compile-remove(teams-inline-images) */
+            onFetchAttachments={onRenderInlineAttachment}
```

Comment:
Discussed this with Josh, I think this should be separate and we should have `onFetchAttachments` and `onFetchInlineImages` as the types are just two different. But consider this non-blocking comment we can iterate on in future PRs